File: TP1 - Dart/pessoa.dart
class Pessoa {
  final int _id;
  String _nome;

  Pessoa(this._id, this._nome);

  // Getters
  int get id => this._id;
  String get nome => this._nome;

  // Setters
  set nome(String nome) => this._nome = nome;
}

File: Pessoa/Pessoa.cpp
#include "Pessoa.h"



Pessoa::Pessoa(string nome){
    this->setNome(nome);
    //this->id = Ver vídeo static (construtores, minuto 12)
}

Pessoa::~Pessoa(){}

string Pessoa::getId() const{
    return this->id;
}

string Pessoa::getNome() const{
    return this->nome;
}

void Pessoa::setNome(string nome){
    this->nome = nome;
}
